core/src/main/java/org/mule/processor/IdempotentRedeliveryPolicy.java

```diff
@@ -57,6 +59,7 @@
     private ObjectStore<AtomicInteger> store;
     private LockFactory lockFactory;
     private String idrId;
+    private boolean forceRedelivery;
```

Comment:
```suggestion
    private static final boolean FORCE_REDELIVERY = getBoolean(MULE_FORCE_REDELIVERY);
```
(And change how it's used)